### DISCUSSION TITLE
Is there a way to avoid packet throttles?

### DISCUSSION THREAD

**Discussion by ITEXNIKJS** [PrismarineJS/mineflayer#2719] [Q&A]
My bot requests 4 blocks in front of it and digs them up every tick. After a certain time, the server kicks the bot with an error. Is there a way to avoid this?

**Comment by amoraschi** [comment]
Maybe don't dig 4 blocks every tick, that's 4 blocks every 50 ms

**Reply by ITEXNIKJS** [reply]
Brilliant. And how did I not guess myself.
